packages/opencensus-web-instrumentation-zone/src/on-page-interaction-stop-watch.ts

```diff
@@ -56,6 +60,16 @@ export class OnPageInteractionStopwatch {
     rootSpan.addAttribute(ATTRIBUTE_HTTP_URL, this.data.startLocationHref);
     rootSpan.addAttribute(ATTRIBUTE_HTTP_PATH, this.data.startLocationPath);
     rootSpan.addAttribute(ATTRIBUTE_HTTP_USER_AGENT, navigator.userAgent);
+    const initialLoadSpanContext = getInitialLoadSpanContext();
+    rootSpan.addAttribute(
+      ATTRIBUTE_INITIAL_LOAD_TRACE_ID,
+      initialLoadSpanContext.traceId
+    );
+    rootSpan.addLink(
+      initialLoadSpanContext.traceId,
+      initialLoadSpanContext.spanId,
+      LinkType.CHILD_LINKED_SPAN
```

Comment:
I think I got the type backwards, I think it should be `PARENT_LINKED_SPAN` since the linked span is the current span's parent. 

Relevant doc: https://github.com/census-instrumentation/opencensus-proto/blob/d4d9f6326c6526ba036327dd0237a468de592f04/src/opencensus/proto/trace/v1/trace.proto#L266